docs(contributing): update roles and responsibilities (#12575)
Esri/calcite-design-system: CONTRIBUTING.md

# Contributing

> Esri welcomes contributions from anyone and everyone. Please see Esri's [guidelines for contributing](https://github.com/esri/contributing).

Reviewer: ```suggestion
Esri welcomes contributions from everyone. Please see Esri's [guidelines for contributing](https://github.com/esri/contributing).
```

## Calcite roles and responsibilities

Calcite provides support to two roles that include different responsibilities: [users](#users) and [contributors](#contributors).

### Users

> Anyone can be a user; no permissions are needed to search, comment, provide feedback, or [create](#before-filing-an-issue) issues using a personal [GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github). _Of note, kind emojis 🏆, kudos 👍, and thank you's 🙌 fuel and energize Calcite team members_ 😊.

Reviewer: ```suggestion
Anyone can be a user; no permissions are needed to search, comment, provide feedback, or [create](#before-filing-an-issue) issues using a personal [GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github). _Of note, kind emojis 🏆, kudos 👍, and thank-yous 🙌 fuel and energize Calcite team members_ 😊.
```

I'm not 100% sure about the hyphen 😄 but I think the apostrophe shouldn't be there, which then makes it feel weird so feel free to ignore if it doesn't feel right.

Users can help most by:<a href="#users-help" id="users-help" />

- Adding reactions, feedback, and/or comments to [existing issues](https://github.com/Esri/calcite-design-system/issues) 👍
- Searching and exploring existing issues, including exploration of the [before filing an issue](#before-filing-an-issue) section prior to filing a new issue
- Reporting issues by [filing a bug issue](https://github.com/Esri/calcite-design-system/issues/new?assignees=&labels=bug%2C0+-+new%2Cneeds+triage&template=bug.yml) 🐛
- Requesting features for existing components by [creating an enhancement issue](https://github.com/Esri/calcite-design-system/issues/new?assignees=&labels=enhancement%2C0+-+new%2Cneeds+triage&template=enhancement.yml) ⭐
- Adding ideas for components by [creating a new component issue](https://github.com/Esri/calcite-design-system/issues/new?assignees=&labels=new+component%2C0+-+new%2Cneeds+triage&template=new-component.yml) 🆕

### Contributors

> Contributors require additional permissions to gain access to the Chromatic test integration suite, where visual changes can be reviewed by the team prior to merging PR's.

Reviewer: ```suggestion
Contributors require additional permissions to gain access to the Chromatic test integration suite, where visual changes can be reviewed by the team prior to merging PRs.
```

To become a contributor, you must:

- Be a current Esri employee
- Be a member of the [Esri](https://github.com/Esri) GitHub organization
  - For access, reach out to the Esri GitHub administrators email alias
- Be a member of the [Calcite Contributors GitHub team](https://github.com/orgs/Esri/teams/calcite-contributors)
  - For access, reach out to [Kitty Hurley](https://github.com/geospatialem) and/or [Juan Carlos Franco](https://github.com/jcfranco)

> Once access is granted to the [Calcite Contributors GitHub team](https://github.com/orgs/Esri/teams/calcite-contributors/members), contributors can [clone the cacite-design-system](https://github.com/Esri/calcite-design-system) repo using an SSH key on their machine, which is the required Git workflow with Calcite's Chromatic test integration suite. Explore the [getting a development environment](#getting-a-development-environment-set-up) section to setup yuor local environment. Contributors should also review the [visual snapshots](#visual-snapshots) section before submitting a PR to familiarize themselves with the Chromatic test integration suite.

Reviewer: ```suggestion
Once access is granted to the [Calcite Contributors GitHub team](https://github.com/orgs/Esri/teams/calcite-contributors/members), contributors can [clone the calcite-design-system](https://github.com/Esri/calcite-design-system) repo using an SSH key on their machine, which is the required Git workflow with Calcite's Chromatic test integration suite. Explore the [getting a development environment](#getting-a-development-environment-set-up) section to setup your local environment. Contributors should also review the [visual snapshots](#visual-snapshots) section before submitting a PR to familiarize themselves with the Chromatic test integration suite.
```

Author: Spelling is hard. 😆 Thanks for these fixes! 🙌🏻

#### I want to contribute, what should I work on?

Contributors can help most by:

- Any of the items listed under the [user role](#users-help)
- Working on [the issues marked as `help wanted`](https://github.com/Esri/calcite-design-system/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee). There is also a [`good first issue`](https://github.com/Esri/calcite-design-system/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee+) label if you are just getting started
  - To let us know of your interest in the issue, please comment on the issue and ask for the action items before you start working. Sometimes additional context is needed, which may not be specified in the issue. Comments also provide us access to assign the issue to you

> - If you want to help develop components take a look at the [new component issues](https://github.com/Esri/calcite-design-system/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+component%22). Before starting development please review our [component conventions](packages/calcite-components/conventions/README.md) and the [Lit documentation](https://lit.dev/docs/getting-started/)

Reviewer: ```suggestion
- If you want to help develop components, take a look at the [new component issues](https://github.com/Esri/calcite-design-system/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+component%22). Before starting development please review our [component conventions](packages/calcite-components/conventions/README.md) and the [Lit documentation](https://lit.dev/docs/getting-started/)
```
